SS_write_report.tpl

```diff
@@ -3297,50 +3297,50 @@ FUNCTION void write_bigoutput()
   {
     SS2out << endl
            << pick_report_name(43) << endl;
-    SS2out << "Method: " << natM_type << endl
-           << "Bio_Pattern Sex Settlement Seas " << age_vector << endl;
-    g = 0;
-    for (gg = 1; gg <= gender; gg++)
-      for (gp = 1; gp <= N_GP; gp++)
-        for (settle = 1; settle <= N_settle_timings; settle++)
-        {
-          g++;
-          if (use_morph(g) > 0)
-          {
-            for (s = 1; s <= nseas; s++)
-              SS2out << gp << " " << gg << " " << settle << " " << s << " " << natM(s, g) << endl;
-          }
-        }
-
-    SS2out << "#" << endl
-           << "Natural_Mortality_Bmark" << endl
-           << "Bio_Pattern Sex Settlement Seas " << age_vector << endl;
-    g = 0;
-    for (gg = 1; gg <= gender; gg++)
-      for (gp = 1; gp <= N_GP; gp++)
-        for (settle = 1; settle <= N_settle_timings; settle++)
-        {
-          g++;
-          if (use_morph(g) > 0)
-          {
-            for (s = 1; s <= nseas; s++)
-              SS2out << gp << " " << gg << " " << settle << " " << s << " " << natM_unf(s, g) << endl;
-          }
-        }
-
-    SS2out << "#" << endl
-           << "Natural_Mortality_endyr" << endl
-           << "Bio_Pattern Sex Settlement Seas " << age_vector << endl;
-    g = 0;
-    for (gg = 1; gg <= gender; gg++)
-      for (gp = 1; gp <= N_GP; gp++)
-        for (settle = 1; settle <= N_settle_timings; settle++)
-        {
-          g++;
-          if (use_morph(g) > 0)
-          {
-            for (s = 1; s <= nseas; s++)
-              SS2out << gp << " " << gg << " " << settle << " " << s << " " << natM_endyr(s, g) << endl;
+    SS2out << "Method: " << natM_type << endl;
+    int kkk = 1;
+    if(N_pred>0)
+    {
+      SS2out<< "area 0 is shows M1 only, numbered areas have M1+M2"<<endl;
```

Comment:
remove "is", I think, to make this statement grammatically correct. Also, Are we using M1 and M2 throughout? I thought we were using something like M and M_pred ?? It would be good to be consistent.